companion/src/firmwareinterface.cpp

```diff
@@ -39,6 +39,8 @@ int getFileType(const QString &fullFileName)
     return FILE_TYPE_EEPE;
   else if (suffix == "XML")
     return FILE_TYPE_XML;
+  else if (suffix == "EEPE2")
```

Comment:
Yes - been waiting for the occasion to get rid of the eepe extension for about 4 years :)